www/shared/styles/imports/variables.css

```diff
@@ -0,0 +1,68 @@
+@import "./custom-medias";
+
+/* ==========================================================================
+   Colors
+   ========================================================================== */
+
+:root {
+    --color-white: #fff;
+}
+
+/* ==========================================================================
+   Typography
+   ========================================================================== */
+
+:root {
+    --font-montserrat: "Montserrat";
```

Comment:
To be honest, I'm not a big fan of this. In my opinion, all typography variants should have names defined in typography and used as mixins. By not having variables here, teams are forced to standardize their typography styles.

See https://github.com/ipfs-shipyard/nomios-web-uikit/blob/master/src/styles/mixins/typography.css for an example of what I mean by standardized typography names.

Reply:
The boilerplate already had this variable :P 
But I'm also a fan of having like this. 
Also, I add on `.body` the default font, so I call this variable. 

The same way the teams are forced to change the mixins, here they are forced to change the name of the variable. 
Neither way,  each team needs to standardize their typography styles in each case.